docs/troubleshooting/reindex-elasticsearch.md

```diff
@@ -24,8 +23,138 @@ This will point Cumulus to the index in Elasticsearch that will be used when ret
 
 The reindex operation will take the data from one index and copy it into another index. The reindex operation can be found [here](https://nasa.github.io/cumulus-api/#reindex)
 
+### Reindex status
```

Comment:
Super small nit/thought: I'm wondering if this could be an addition to the `Reindex from the old index to the new index` sub-heading above rather than a separate section? I like that the main section, `Switch to a new index and Reindex` has two sub-sections that are explicitly called out in the first sentence. That section would read:

**Reindex from the old index to the new index**
The reindex operation will take the data from one index and copy it into another index. The reindex operation can be found here. Reindexing is a long-running operation. The reindex-status endpoint can be used to monitor the progress of the operation.

Reply:
Ah yep that makes sense. I made that change.